### ISSUE TITLE
Duplicate queries selecting from form table on every request.

### ISSUE THREAD

**Issue by Shelob9** [CalderaWP/Caldera-Forms#3553]
## What Version Of Caldera Forms, WordPress and PHP Are You Using?
Short Debug Information:

`WordPress Version: 5.4.2 PHP Version: 7.3.18 MySQL Version: 5.5.5 Caldera Forms Version:   1.8.11`

Currently Active Theme: Twenty Twenty: 1.2
Akismet Anti-Spam: 4.1.5 Not Active
Caldera Forms: 1.8.11 Active
Hello Dolly: 1.7.2 Not Active
Query Monitor: 3.6.0 Active
## Does Your Issue Persist When You Disable All Other Plugins and Switch To The Default Theme?
Yes.

## What Is The Unexpected Behaviour?

I read these reports: 
https://wordpress.org/support/topic/duplicate-queries-10/ and I looked at CalderaForms.com new relic, which shows wp_cf_forms select as slowest query for last 7 days and last 60 minutes.

On  site with no forms, on front-end page with no forms, I see a duplicate query in Query monitor from `wp_cf_forms`

![dupes](https://user-images.githubusercontent.com/1994311/84914341-7ea56c80-b089-11ea-8f3d-2c493cd4fb83.jpg)

Adding a form changes the duplicate query

![dupes-2](https://user-images.githubusercontent.com/1994311/84914804-0db28480-b08a-11ea-9ee9-d9acd865567f.jpg)

Thriplicated queries on the page with form:

![image](https://user-images.githubusercontent.com/1994311/84914992-3cc8f600-b08a-11ea-9286-b90c618c94da.png)

## What PHP Errors Have You Logged While Reproducing This Bug?
None

## What JavaScript Errors Have You Seen While Reproducing This Bug?
none

**Comment by Shelob9** [comment]
These properties, used to power a cache to avoid these duplicate queries. https://github.com/CalderaWP/Caldera-Forms/blob/master/classes/forms.php#L19-L44



Related: 

- https://github.com/CalderaWP/Caldera-Forms/commit/1a6a1e0c69fa2152df0e64e5ac8dc2c8e937ba88#diff-634fa6588d7d78d435ea1ea46e18563cR418
- #3354
- #3341

**Comment by Shelob9** [comment]
I spent some more time with query monitor. Many different combos of functions end up calling `Caldera_Forms_DB_Form::get_instance()->get_all();`, which does the offending query.

[There is only one place that function is called in the codebase](https://github.com/CalderaWP/Caldera-Forms/blob/master/classes/forms.php#L246). I checked using phpstorm, it's only being used there and in test files:
![image](https://user-images.githubusercontent.com/1994311/85303107-0fa88900-b478-11ea-8c27-10cca741f710.png)

How this should work is 
* The first time we need the query, the property $stored_forms is empty, so 245 evaluates true.
     * https://github.com/CalderaWP/Caldera-Forms/blob/master/classes/forms.php#L245
* The we populate $stored_forms beacuse 249 is true.
     * https://github.com/CalderaWP/Caldera-Forms/blob/master/classes/forms.php#L249
* Function is called again and 245 is false, so we always return the $stored_forms variable, no dupe queries.

One or more of these points is untrue.
